apps/docs/docs/quickstart.mdx

```diff
@@ -3,13 +3,29 @@ import TabItem from '@theme/TabItem';
 
 # Quickstart
 
+Setup is easy. 
+
+Once you understand how to use `ts-rest`, developer experience is like butter.  🧈
+
 ## Installation
 
-Install the core package
+1. Install the core package
 
 ```bash
-yarn add @ts-rest/core
+yarn add @ts-rest/core 
+--
+npm i @ts-rest/core
+```
```

Comment:
No pnpm docs even though we are using pnpm 😆

Reply:
@rifaldhiaw had a PR for this #165 which I just merged, might have to fix some conflicts sorry @michaelangrivera !

Reply:
Hah, they out did me! @rifaldhiaw